runhouse/resources/hardware/cluster.py

```diff
@@ -115,6 +117,11 @@ def __init__(
         if self._default_env and not self._default_env.name:
             self._default_env.name = _unnamed_default_env_name(self.name)
 
+        disable_observability = kwargs.get(
```

Comment:
does this actually flow thru? i think we may need to add this to the `alt_options`?